src/main/java/com/example/nlern/models/FullTest.java

```diff
@@ -0,0 +1,14 @@
+package com.example.nlern.models;
+
+import lombok.Getter;
+import lombok.Setter;
+import java.util.List;
+
+@Getter @Setter
+public class FullTest {
+    Test test;
+    List<Question> questions;
+    List<Integer> deleted;
+    List<Integer> added ;
```

Comment:
You have not fixed spacing everywhere. IIRC IntelliJ should handle this kind of refactoring when using "Reformat code" (https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html), you should not fix this manually

Reply:
thanks, this is cool feature)